src/main/java/team/silvertown/masil/user/domain/User.java

```diff
@@ -109,4 +112,26 @@ public void toggleIsPublic() {
         this.isPublic = !this.isPublic;
     }
 
+    public void updateStats(int distance, int calories) {
+        initializeStats();
+
+        this.totalDistance += distance;
+        this.totalCalories += calories;
+        this.totalCount++;
+    }
+
+    private void initializeStats() {
+        if (Objects.isNull(this.totalDistance)) {
+            this.totalDistance = 0;
+        }
+
+        if (Objects.isNull(this.totalCount)) {
+            this.totalCount = 0;
+        }
+
+        if (Objects.isNull(this.totalCalories)) {
+            this.totalCalories = 0;
+        }
+    }
```

Comment:
세 값 중에 하나라도 null 이라면 모든 필드를 0으로 초기화하는게 좋아보이는데요?
```suggestion
        if (hasNullStats()) {
            this.totalDistance = 0;
            this.totalCount = 0;
            this.totalCalories = 0;
        }
    }
```